droid-hal-device.inc

```diff
@@ -607,6 +616,14 @@ if [ $android_version_major -ge "10" ]; then
     # So we have to build apex (make com.android.runtime.release) and then:
     art_path=%{android_root}/out/soong/.intermediates/art/build
     apex_path=apex/com.android.runtime.release/android_common_com.android.runtime.release/image.apex
+    if [ ! -f "$art_path/$apex_path/lib/bionic/libc.so" ]; then
+        art_path=%{android_root}/out/target/product/%{device}/system
+        apex_path=apex/com.android.runtime.release
+    fi
```

Comment:
This has already appeared in another PR that only needs merging: https://github.com/mer-hybris/droid-hal-device/pull/305/files#diff-fa405a547d9e4b07d7be494e40c5135280c293ec1084662360864ae64a6aca68R611

Reply:
Rebased after merging the other PR.